src/box/lua/config/applier/box_cfg.lua

```diff
@@ -496,14 +496,35 @@ local function apply(config)
         --
         -- The RO/RW mode remains unchanged on reload.
         local am_i_bootstrap_leader = false
+        local recently_bootstrapped = false
+
         if is_startup then
             local instance_name = configdata:names().instance_name
+
+            recently_bootstrapped =
+                snapshot.get_path(configdata._iconfig_def) == nil 
+
             am_i_bootstrap_leader =
-                snapshot.get_path(configdata._iconfig_def) == nil and
+                recently_bootstrapped and
                 instance_name == configdata:bootstrap_leader_name()
+
             box_cfg.read_only = not am_i_bootstrap_leader
         end
 
+        if recently_bootstrapped then
+            local renew_interval =
+                instance_config:get(configdata._iconfig_def,
+                                    'failover.renew_interval')
```

Comment:
The coordinator grants RW for `failover.lease_interval`. I would do it here too.

Reply:
You're right. Thank you. Done.